README.md

```diff
@@ -38,7 +38,7 @@ git clone git@github.com:jelhan/croodle.git
 cd croodle
 yarn install
 cd api/ && composer install --no-dev && cd ..
-ember build --prod
+yarn build -- --prod
```

Comment:
For what case are the ` -- ` needed?

```suggestion
yarn build --prod
```

Reply:
I thought it's needed maybe it's just needed for npm.

I tested it and it works as you suggested :+1: